src/BulletSoftBody/btSparseSDF.h

```diff
@@ -103,7 +103,7 @@ struct btSparseSdf
 				pc = pn;
 			}
 		}
-		voxelsz = 0.25;
+		voxelsz = 0.025;
```

Comment:
please undo default change, and move assignment of voxelssz in your code (who needs 0.025)

Reply:
Undid the default change.